.github/workflows/eslint.yml

```diff
@@ -3,7 +3,7 @@
 # More details at https://github.com/eslint/eslint
 # and https://eslint.org
 
-name: ESLint
```

Comment:
Can you undo this change? There will be multiple linters, so it is better to be explicit

Reply:
@ermish done